cmd/csi-attacher/main.go

```diff
@@ -67,30 +67,31 @@ var (
 
 func main() {
 	flag.Set("logtostderr", "true")
+	klog.InitFlags(nil)
```

Comment:
Hmm, does `flag.Set("logtostderr", "true")` work when it comes before the definition of the `logtostderr` flag itself in `klog.InitFlags(nil)`?

Reply:
It doesn't :-(

Reply:
No, it doesn't, I tried. `klog.InitFlags(nil) must come first.

Reply:
It's fixed now.